jbpm-services/jbpm-kie-services/src/main/resources/META-INF/Servicesorm.xml

```diff
@@ -698,86 +698,88 @@
     <!-- hint name="org.hibernate.timeout" value="200"/ -->
   </named-query>
   
-  <named-query name="getTaskInstanceById">
+  <named-native-query name="getTaskInstanceById">
     <query>
         select 
-	      new org.jbpm.kie.services.impl.model.UserTaskInstanceDesc(
```

Comment:
why this you remove this ? It is better to retrieve the custom object UserTaskInstance (type safe)  that an array of objects. The transformation should be the other way round.